ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,8 +429,10 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
+            worker_type=5.4xlarge
             if [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
               master_type=m5.8xlarge
+              worker_type=m5.8xlarge
```

Comment:
my suggestion was to introduce a new variant type that only sets the worker_type.  If you do this, all the existing jobs that use the "xlarge" variant are going to get xlarge workers and xlarge masters, which isn't what they need.